pyopenms_viz/_config.py

```
@@ -1,15 +1,346 @@
 from abc import ABC, abstractmethod
-from dataclasses import dataclass, field
-from typing import Tuple, Literal, Dict, Any
+from dataclasses import dataclass, field, asdict, fields
+from typing import Tuple, Literal, Dict, Any, Union, Iterator
 from enum import Enum
```

Ruff (0.8.2)

`enum.Enum` imported but unused

Remove unused import: `enum.Enum`

(F401)